<html lang="en-US">
 <body>
  <h2 id="	">            
    <ul>
    <font size="8">Protocol:<a href="https://github.com/yaqike/loquat/blob/main/lab-pipeline.sh">Pipeline</font></a>
    <h5 align="left"> </h5>
      <ul>
      <li>QIIME 2&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/qiime2">Example Datasets</a></li>
      <font size="3">[1]Bolyen, Evan et al. “Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2.” Nature biotechnology vol. 37,8 (2019): 852-857.</font>
      <h3 align="left"> </h3>

      <li>PICRUSt2&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/picrust2">Example Datasets</a></li>
      <font size="3">[2]Douglas, Gavin M et al. “PICRUSt2 for prediction of metagenome functions.” Nature biotechnology vol. 38,6 (2020): 685-688.</font>
      <h3 align="left"> </h3>

      <li>MicrobiomeAnalyst&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/MicrobiomeAnalyst">Example Datasets</a></li>
      <font size="3">[3]Chong, Jasmine et al. “Using MicrobiomeAnalyst for comprehensive statistical, functional, and meta-analysis of microbiome data.” Nature protocols vol. 15,3 (2020): 799-821.</font>
      <h3 align="left"> </h3>
       
      <li>ggClusterNet&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/ggClusterNet">Example Datasets</a></li>
      <font size="3">[4]“ggClusterNet: An R package for microbiome network analysis and modularity-based multiple network layouts.” iMeta https://doi.org/10.1002/imt2.32</font>
      <h3 align="left"> </h3>
      </ul>
    </ul>
    <br>
    <br>
    <h5 align="center">Powered by YAqiKe</h5> 
 </body>
</html>
